src/components/PlantsV2/index.tsx

```diff
@@ -55,16 +55,15 @@ export default function PlantsDashboardV2(): JSX.Element {
   );
 
   useEffect(() => {
-    dispatch(requestPlantingSites(org.selectedOrganization.id));
     dispatch(requestObservations(org.selectedOrganization.id));
     dispatch(requestObservationsResults(org.selectedOrganization.id));
     dispatch(requestSpecies(org.selectedOrganization.id));
-  }, [dispatch, org]);
+    dispatch(requestPlantings(org.selectedOrganization.id));
```

Comment:
fetch once for all sites in org, we use selectors to select data for specific sites